extensions/xchange/deployment/src/main/java/org/apache/camel/quarkus/component/xchange/deployment/XchangeProcessor.java

```diff
@@ -138,7 +138,7 @@ void registerProxyClasses(BuildProducer<NativeImageProxyDefinitionBuildItem> nat
 
     @BuildStep
     void runtimeInitializedClasses(BuildProducer<RuntimeInitializedClassBuildItem> runtimeInitializedClasses) {
-        Stream.of(Order.class.getName())
+        Stream.of(Order.class.getName(), "org.apache.http.impl.auth.NTLMEngineImpl")
```

Comment:
Is this definitely required? I don't see the Apache HTTP client on the runtime classpath for camel-xchange.